includes/Ajax.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Wedevs\DokanMigrator;
+
+use Exception;
+use Wedevs\DokanMigrator\Helpers\MigrationHelper;
+
+/**
+ * Ajax request handler class.
+ *
+ * @since 1.0.0
+ */
+class Ajax {
+
+    /**
+     * Class constructor.
+     *
+     * @since 1.0.0
+     */
+    public function __construct() {
+        add_action( 'wp_ajax_dokan_migrator_count_data', array( $this, 'count' ) );
+        add_action( 'wp_ajax_dokan_migrator_import_data', array( $this, 'import' ) );
+        add_action( 'wp_ajax_dokan_migrator_last_migrated', array( MigrationHelper::class, 'get_last_migrated' ) );
+        add_action( 'wp_ajax_dokan_migrator_active_vendor_dashboard', array( MigrationHelper::class, 'active_vendor_dashboard' ) );
+    }
+
+    /**
+     * Count the data of vendor or order.
+     *
+     * @since 1.0.0
+     *
+     * @return void
+     */
+    public function count() {
+        $this->verify_nonce( $_REQUEST ); // phpcs:ignore
```

Comment:
Instead of using `// phpcs:ignore`, make the declaration more specific mentioning which rule you're ignoring. For example, `// phpcs:ignore Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed`